scripts/PopUpMessage.js

```diff
@@ -0,0 +1,12 @@
+// Copy Notification
+class PopUpMessage {
+  constructor(msg) {
+    this.element = document.createElement('div');
+    this.element.className = 'message';
+    this.element.innerHTML = 'Color '+ msg + ' is copied to clipboard!';
+  }
+}
+
+module.exports = {
```

Comment:
If you have only one something that you export - write just module.exports = PopUpMessage
And change imports for this approach after exports changes